frontend/tsconfig.json

```diff
@@ -25,5 +29,5 @@
     "skipLibCheck": true
   },
   "exclude": ["node_modules", "build"],
-  "include": ["src/**/*.ts", "src/**/*.tsx", "declarations/**/*.d.ts"]
+  "include": ["src", "declarations/**/*.d.ts"]
```

Comment:
Is there a reason for this change? Im assuming it actually just simplifies the complifier config without changing anything? As in Css files are ignored, and svgs are ignored.

Reply:
yes, just simplification